### PR TITLE
Create a new branch to avoid conflicts

### PR DESCRIPTION
**Pull request by Fred1110**
Created a new branch for companies to avoid conflicts.
In seed file the 'assert' identifier does not work. On stackoverflow there are people saying the 'assert' is not supported by npm anymore so I changed it to 'with'.

### CODE REVIEW

**Comment by jtshu** [comment]
approved from my end -- just needs one more approver before we can merge as per our team agreement

**Review by jphalis — APPROVED** [review]

**Pull request merged**
